src/Sphere.jl

```diff
@@ -5,12 +5,13 @@ end
 project_tangent!(S::Sphere, w, x, v) = (w .= v .- dot(x, v).*x)
 distance(S::Sphere, x, y) = acos(dot(x, y))
 
-function exp!(S::Sphere, y, x, v)
-    nv = norm(S, x, v)
+function exp!(S::Sphere, y, x, v, t=1)
+    scaled_v = t*v
```

Comment:
This doesn't seem necessary, since we then just compute the norm of `v`. What about

```julia
nv = norm(S, c, v)
tnv = t * nv
    if tnv ≈ 0.0
        y .= x
    else
        y .= cos(tnv).*x .+ (sin(tnv)/nv).*v
    end
    return y
end
```

Reply:
The other change will make this not necessary.